src/components/selection.ts

```diff
@@ -267,6 +267,28 @@ export default class SelectionUtils {
     this.savedSelectionRange = null;
   }
 
+  /**
+   * Clear current window selection
+   */
+  public clearWindowSelection(): void {
+    const sel = window.getSelection();
+    const range = document.createRange();
+    /**
+     * Set node content to node where selection ends
+     */
+    range.selectNodeContents(sel.focusNode);
```

Comment:
I think `clearWindowSelection` should not contain Range restoration functionality, because It might be used separately. Probably, it is better to move this logic to another method and add to this function optional argument `restoreRange` which is `false` by default? And then according to the value of this flag call additional function

Reply:
Selection has API for collapse to END, i'm trying that. I might not need clear window selection method.

Reply:
Are you sure about browsers compatibility? https://developer.mozilla.org/en-US/docs/Web/API/Selection/collapseToEnd

Reply:
Yeah, `collapseToEnd` is marked as experimental. So I replaced it.

Reply:
Good